composer.json

```diff
@@ -11,18 +11,18 @@
         { "name": "Knplabs", "homepage": "http://knplabs.com" }
     ],
     "require": {
-        "php": ">=8.0",
+        "php": ">=8.1",
```

Comment:
This should be reverted as Symfony 6 still supports PHP 8.0 unless there is a reason for it.